ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml

```diff
@@ -15894,12 +15894,12 @@ periodics:
     repo: openshift-tests-private
   labels:
     ci-operator.openshift.io/cloud: vsphere
-    ci-operator.openshift.io/cloud-cluster-profile: vsphere
+    ci-operator.openshift.io/cloud-cluster-profile: vsphere-discon
     ci-operator.openshift.io/variant: amd64-nightly
     ci.openshift.io/generator: prowgen
     job-release: "4.13"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-vsphere-upi-ovn-sdn2ovn-p2-f7
```

Comment:
Should also remove ` periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-azure-ipi-ovn-sdn2ovn-p2-f7`?

Reply:
fixed.